src/stories/epics/progress/index.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import LinkTo from '@storybook/addon-links/react';
+import EpicProgress from '~js/components/epics/progress';
+
+<Meta title="Epics/Progress/Docs" />
+
+# Epic Progress
+
+**Visualize the overall progress/status of the entire project.**
+
+## Overview
+
+The **Epic Progress** component provides a quick view of the progress when
```

Comment:
Maybe: "The **Epic Progress** component visualizes the progress made on **Tasks** within an **Epic**." ?